code/modules/projectiles/guns/ds13/divet.dm

```diff
@@ -138,20 +134,13 @@ Projectile logic
 	armor_penetration = 15
 	icon_state = "divet_ap"
 
+//Mostly normal rounds with a little extra armor pen, but they also set you on fire
 /obj/item/projectile/bullet/ls_slug/incendiary
-	damage_type = BURN
 	icon_state = "divet_incend"
-	fire_sound = list('sound/weapons/guns/fire/torch_altfire_1.ogg',
-	'sound/weapons/guns/fire/torch_altfire_2.ogg',
-	'sound/weapons/guns/fire/torch_altfire_3.ogg')
-
+	armor_penetration = 9
 
-/obj/item/projectile/bullet/ls_slug/incendiary/on_impact(atom/target)
```

Comment:
this is an important feature of them, don't remove the fire

Reply:
the temperature, radius, and duration of the fire effect can be tweaked to adjust its power.

fire being available is fairly important as its one of the very few sources for crew to access flame weaponry that can fight against corruption, this is very much an anti-necromorph bullet

Reply:
The replacement system switches the damage type to brute and lights the target on fire.